react-native-libraries.json

```diff
@@ -10079,5 +10079,29 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/farhoudshapouran/react-native-ui-datepicker",
+    "npmPkg": "react-native-ui-datepicker",
+    "examples": ["https://github.com/farhoudshapouran/react-native-ui-datepicker/tree/main/example"],
+    "images": [
+      "https://user-images.githubusercontent.com/7857656/227187674-93012672-495d-4955-b4d3-46c3d016684e.jpg",
+      "https://user-images.githubusercontent.com/7857656/227260476-30ee8c25-f809-4dcf-bccf-cd1ffab8795a.jpg",
+      "https://github.com/farhoudshapouran/react-native-ui-datepicker/raw/main/.github/images/react-native-ui-datepicker-example.gif"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/farhoudshapouran/react-native-highlighter",
+    "npmPkg": "react-native-highlighter",
+    "examples": ["https://github.com/farhoudshapouran/react-native-highlighter/tree/main/example"],
+    "images": ["https://user-images.githubusercontent.com/7857656/227882432-2d18c750-5c96-4838-8dfd-62bd44cfa18f.jpg"],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expo": true
   }
```

Comment:
Hello @farhoudshapouran, thank you for adding new entries to the directory! 🙏 

Let's remove `npmPkg` fields, since they are mostly dedicated for scoped packages, and for regular packages the repository name will be used (so they are skippable, if repo name matches package name).

```suggestion
  {
    "githubUrl": "https://github.com/farhoudshapouran/react-native-ui-datepicker",
    "examples": ["https://github.com/farhoudshapouran/react-native-ui-datepicker/tree/main/example"],
    "images": [
      "https://user-images.githubusercontent.com/7857656/227187674-93012672-495d-4955-b4d3-46c3d016684e.jpg",
      "https://user-images.githubusercontent.com/7857656/227260476-30ee8c25-f809-4dcf-bccf-cd1ffab8795a.jpg",
      "https://github.com/farhoudshapouran/react-native-ui-datepicker/raw/main/.github/images/react-native-ui-datepicker-example.gif"
    ],
    "ios": true,
    "android": true,
    "web": true,
    "expo": true
  },
  {
    "githubUrl": "https://github.com/farhoudshapouran/react-native-highlighter",
    "examples": ["https://github.com/farhoudshapouran/react-native-highlighter/tree/main/example"],
    "images": ["https://user-images.githubusercontent.com/7857656/227882432-2d18c750-5c96-4838-8dfd-62bd44cfa18f.jpg"],
    "ios": true,
    "android": true,
    "web": true,
    "expo": true
  }
```

besides that small nit, LGTM! 👍